runbot_gitlab/models/runbot_repo.py

```diff
@@ -151,23 +151,25 @@ class RunbotRepo(models.Model):
     @api.model
     def create(self, vals):
         repo_id = super(RunbotRepo, self).create(vals)
-        set_gitlab_ci_conf(
-            vals.get('token'),
-            vals.get('name'),
-            self.domain(),
-            repo_id.id,
-        )
+        if vals.get('uses_gitlab', False):
```

Comment:
You should be able to just check `repo_id.uses_gitlab` since vals have already created in super.